packages/plugins/experimental/plugin-map/src/hooks/useMarkers.ts

```diff
@@ -66,13 +43,13 @@ export const useMarkers = (map: MapType): MapMarker[] => {
           lng: row.longitude,
         },
       }));
-  }, [objects]);
+  }, [JSON.stringify(objects.map((obj) => obj.id))]); // TODO(seagreen): Hack.
```

Comment:
This way adding non-location ECHO objects doesn't trigger a change to our returned object reference.

This might not be great, but it's not new, previously we did the deleted line above:

> [JSON.stringify(Array.from(schemaIds))], // TODO(burdon): Hack.

Reply:
ok to merge, but pls ask @dmaretskyi about this.

Reply:
Done!